src/ng/directive/ngOptions.js

```diff
@@ -420,12 +420,19 @@ var ngOptionsDirective = ['$compile', '$document', '$parse', function($compile,
 
       // The emptyOption allows the application developer to provide their own custom "empty"
       // option when the viewValue does not match any of the option values.
-      for (var i = 0, children = selectElement.children(), ii = children.length; i < ii; i++) {
-        if (children[i].value === '') {
-          selectCtrl.hasEmptyOption = true;
-          selectCtrl.emptyOption = children.eq(i);
-          break;
-        }
+      var optionsToRemove = [];
+      for (var i = 0, children = selectElement[0].childNodes, ii = children.length; i < ii; i++) {
```

Comment:
Is the move from jqLite/jQuery APIs to native APIs part of the perf fix or is it just a "generally good idea" type of change?